src/project-tests/pomodoro-clock-tests.js

```diff
@@ -78,476 +84,718 @@ export default function createPomodoroClockTests() {
+              reject(
+                new Error('Test timed out because Break time didn\'t start ' +
+                'with the correct value: ' + (
+                  parseInt(getMinutes(timeLeft.innerHTML), 10) + 1
+                ) + ' instead of ' + breakLength.innerHTML)
+              );
+            } else if (timeLeft.innerHTML === '00:00') {
+              // note: sound has to be longer than 200 ms, or the test will 
+              // fail if the sound stops before the test actually happens
+              savedSetTimeout(() => {
+                const audioElem = document.getElementById('beep');
+
+                if (audioElem && !audioElem.paused) {
                   resolve();
                 } else {
-                  reject(new Error("Timer has reached zero but, either there is not audio tag with ID 'beep' on the page, or it's not playing while it should."));
+                  reject(
+                    new Error('Timer has reached zero but, either there is ' +
+                    'not audio tag with ID "beep" on the page, or it\'s not ' +
+                    'playing while it should.')
+                  );
                 }
               }, 200);
+              observer.disconnect();
             }
           });
         });
       });
 
-      it(`27. The audio element with id="beep" must be 1 second or longer.`,
-      function() {
-        FCC_Global.assert.isAbove(
+      reqNum++;
+      it(`${reqNum}. The audio element with id="beep" must be 1 second or 
+      longer.`, function() {
+        assert.isAbove(
           document.getElementById('beep').duration,
           1,
-          `Audio element with id="beep" is not at least 1 second long.`
+          'Audio element with id="beep" is not at least 1 second long.'
         );
       });
 
-      it(`28. The audio element with id of "beep" must stop playing and be
-      rewound to the beginning when the element with the id of "reset" is
+      reqNum++;
+      it(`${reqNum}. The audio element with id of "beep" must stop playing and 
+      be rewound to the beginning when the element with the id of "reset" is
       clicked.`,
       function() {
         var audioElem = document.getElementById('beep');
 
         audioElem.play();
         resetTimer();
 
-        FCC_Global.assert.isTrue(
+        assert.isTrue(
           audioElem.paused,
-          `Audio element was not stopped when reset was clicked.`
+          'Audio element was not stopped when reset was clicked.'
         );
 
-        FCC_Global.assert.equal(
+        assert.equal(
           0,
           audioElem.currentTime,
-          `Audio element was not rewound when reset was clicked. HINT: use the
-          currentTime property of the audio element to rewind.`
+          'Audio element was not rewound when reset was clicked. HINT: use ' +
+          'the currentTime property of the audio element to rewind.'
         );
       });
 
-    }); // END #Tests
+    });
+    // END #Tests
```

Comment:
Could you please put this comment *above* the closing bracket & paren? Not that one is any better than the other, but I have been putting them above, so we should be consistent. Same for the comments below.